src/host/ut_host/InputBufferTests.cpp

```diff
@@ -611,6 +611,7 @@ class InputBufferTests
         auto inputEvent2 = IInputEvent::Create(record2);
         // write another event to a non-empty buffer
         waitEvent = false;
+        storage.clear();

```

Comment:
Starting with this PR, `_WriteBuffer` moves its items out of the argument, without emptying the argument. This results in this test failing. In a follow-up PR this will not be a problem anymore, because I'll remove `IInputEvent`.